tests/test_remote_inventory.py

```diff
@@ -0,0 +1,113 @@
+import os
+import sys
+import unittest
+import tempfile
+from shutil import rmtree, copytree
+from distutils.dir_util import copy_tree
+from kapitan.cached import reset_cache
+from kapitan.cli import main
+from kapitan.remoteinventory.fetch import (
+    fetch_git_source,
+    fetch_git_inventories,
+    fetch_http_source,
+    fetch_http_inventories,
+)
+from kapitan.dependency_manager.base import DEPENDENCY_OUTPUT_CONFIG
+
+
+class RemoteInventoryTest(unittest.TestCase):
+    def setUp(self):
+        os.chdir(
+            os.path.join(
+                os.path.abspath(os.path.dirname(__file__)), "test_remote_inventory", "environment_one"
+            )
+        )
+
+    def test_fetch_git_inventory(self):
+        temp_dir = tempfile.mkdtemp()
+        git_source = "https://github.com/deepmind/kapitan.git"
+        fetch_git_source(git_source, temp_dir)
+        self.assertTrue(os.path.isdir(os.path.join(temp_dir, "kapitan.git", "kapitan")))
```

Comment:
Can we check here for a file instead? Maybe the README.md one. 
That will always exist and it's slightly safer as a test than checking for the directory since directories could be created in various other scenarios when using git too.